.github/workflows/template_autodev_app.yml

```diff
@@ -0,0 +1,90 @@
+name: Autodev
+on:
+  workflow_call:
+    inputs:
+      base:
+        default: 'main'
+        required: false
+        type: string
+      branch:
+        default: 'dev'
+        required: false
+        type: string
+      comments:
+        default: false
+        required: false
+        type: boolean
+      email:
+        default: 'staffbot@staffbase.com'
+        required: false
+        type: string
+      failure_comment:
+        required: false
+        type: string
+      failure_label:
+        default: 'failed'
+        required: false
+        type: string
+      label:
+        default: 'dev'
+        required: false
+        type: string
+      labels:
+        default: false
+        required: false
+        type: boolean
+      success_comment:
+        required: false
+        type: string
+      success_label:
+        default: 'successful'
+        required: false
+        type: string
+      user:
+        default: 'AutoDev Action'
+        required: false
+        type: string
+    secrets:
+      app_id:
+        required: true
+      private_key:
+        required: true
+
+jobs:
+  autodev:
+
+    name: Build Dev Branch
+    runs-on: ubuntu-22.04
+
+    # check to not trigger if dependabot did something or PR was closed or label is not the configured dev label
+    if: github.actor != 'dependabot[bot]' && (github.event_name == 'push' || github.event.label.name == inputs.label || github.event.action == 'closed')
+
+    steps:
+      - name: Get App Token
```

Comment:
That's exactly what Falk wants to avoid - now the workflow requires a Github App.
The App should be optional. So it should be still possible to use the workflow with a PAT.

Reply:
And I guess this is still possible with the other configuration?! 🤔 But with the two implementations you have now the advantage that we have no complexity issues.

Reply:
But duplication. Can you adjust it to have a single workflow which work for both PAT and App, just to see how it will look like (maybe it will be less complicated).

Reply:
I will try to do it in another PR. 🙌